src/intTest/resources/patient/amendment-empty-patch-object.json

```diff
@@ -0,0 +1,7 @@
+{
+  "nhsNumber": "9999999999",
+  "gpCode": "8880232,G82697",
+  "gpTradingPartnerCode": "TES5",
+  "healthcarePartyCode": "XX1",
+  "patches": [{}]
```

Comment:
This will be `"patches" : [ ]` but no worries, I'll adjust this in my PR

Reply:
This test case is specifically for an array containing and empty object. There is another test for just an empty array.